package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "webpack-setup",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": true,
+  "scripts": {
+    "start": "webpack serve --mode development",
+    "build": "webpack --mode production",
+    "test": "jest"
+  },
```

Comment:
- [ ] Good job adding `jest` to the package.json file. However when a user runs your test, they have errors, one is actually pointing to the `console.log()` STATEMENT IN  `comment.js line 11' Kindly fix all the errors.

![image](https://github.com/ANTOINE1128/javascript-capstone/assets/81932478/37190303-f03f-4d56-b498-09699599030d)